### PR TITLE
Update various JavaScript libraries for security and other fixes

### PR DESCRIPTION
**Pull request by reedloden**
* Update jQuery to latest to fix multiple security vulnerabilities
* Use minified version of jQuery
* Update jquery.twitter.js so https:// is used for twitter.com links
* Update paging_keys.js for bug fixes

### CODE REVIEW

**Comment by reedloden** [comment]
Perhaps I should use the pre-minified versions of jQuery for performance reasons?

**Comment by YorickPeterse** [comment]
To double check, this doesn't break any functionality on the website right?

**Comment by reedloden** [comment]
Not as far as I could tell... I did `bundle exec jekyll serve` and played around with the site, and it seemed to work fine. However, it's possible there's some functionality I'm not sure about.

File: javascripts/paging_keys.js [hunk 1]
```diff
@@ -75,14 +75,14 @@ HotKey.prototype.remove = function(key){
...
-/* by Matthew Hutchinson - hiddenloop.com */
+/* https://github.com/matthutchinson/paging_keys_js - inspired by navigation on ffffound.com UI */
+/* by Matthew Hutchinson - matthutchinson.net */
 
 var pagingKeys = function() {
 
 	// settings
 	var config = {
-    nodeSelector:        '.stripe',  // used to select each item on the page and place in the map (must be a link)
+    nodeSelector:        '.hentry h2 a.entry-title',  // used to select each item on the page and place in the map (must be a link)
```
Comment (YorickPeterse) [inline javascripts/paging_keys.js]: This is the only change I'm unsure about as we might have some CSS depend on it. If not then this should be fine.

Comment (reedloden) [inline javascripts/paging_keys.js]: Ah, indeed. I'll just revert the change to that file, as it was just bug fixes and not security-related.

Comment (YorickPeterse) [inline javascripts/paging_keys.js]: They don't need to be reverted if nothing depends on it.

Comment (reedloden) [inline javascripts/paging_keys.js]: Well, `stylesheets/application.css` does have rules for `.stripe`. I don't see anything else using them, though (nothing seems to actually use that class, besides paging_keys.js trying to select elements based on it).

**Comment by brixen** [comment]
@reedloden thanks!

**Pull request merged**